design/airgap-existing-cluster.md

```diff
@@ -0,0 +1,110 @@
+# Support for installing KOTS Admin Console in Airgapped Clusters
+
+## Goals
+
+- Install kotsadm to an existing cluster where the cluster does not have internet access
+- Install kotsadm to an existing cluster from a workstation where the workstation does not have internet access at the time of installation
+- Support different credentials for push / pull (workstation / cluster) to the local registry
+- Support temporary/dynamic and persistent/static credentials for pushing kotsadm images to the registry
+- Support only persistent/static credentials for pulling kotsadm images to the registry
+- Support different credentials for pushing vs pulling images to the local registry
+
+## Non Goals
+
+- Create a single installer for kotsadm + application
+- Create a single download for kotsadm, kots plugin, application and license
+- Automated installs of kotsadm to existing clusters in airgap mode
+- Support private (authenticated) local hosting of kotsadm images
```

Comment:
I'm not sure what we mean by "local hosting" but this needs to include the ability to store the kotsadm images in my private registry

Reply:
I think that was a leftover from an earlier iteration.
```suggestion
- Support private (authenticated) local hosting of kotsadm images
```